robust/testing/t_primitives.py

```diff
@@ -9,18 +9,6 @@
 from robust.testing.models import sp_test_model, gp_test_model
 
 class TestPrimitives(unittest.TestCase):
-    def test_MarginModel(self):
-        """ Tests creation and solution of MarginModel"""
-        m = sp_test_model()
-        mm = MarginModel(m, gamma = 0.5)
-        margin_solution = mm.localsolve(verbosity=0)
-        uncertain_varkeys = [k for k in m.varkeys if RobustGPTools.is_directly_uncertain(k)]
-        # Checking margin allocation
-        for key in list(uncertain_varkeys):
-            assert(mm.substitutions.get(key) == m.substitutions.get(key) *
-                                                (1.+mm.setting.get("gamma")*key.pr/100.))
-        self.assertGreater(margin_solution['cost'], mm.nominal_cost)
-
```

Comment:
reason for removing?

Reply:
Because it is an exact duplicate of the test under it... it was a merge error.